### DISCUSSION TITLE
Welcome to squeezelite-docker Discussions!

### DISCUSSION THREAD

**Discussion by GioF71** [GioF71/squeezelite-docker#175] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CrimsonAC** [comment]
Hello, i just tried to install this onto an old Raspberry Pi B+ without success. Is it possible to get this image compatible to ARM v6?

**Reply by GioF71** [reply]
> Hello, i just tried to install this onto an old Raspberry Pi B+ without success. Is it possible to get this image compatible to ARM v6?

Resolved with #179
